app/assets/stylesheets/common/components/navs.scss

```diff
@@ -124,3 +124,8 @@
     line-height: var(--line-height-large);
   }
 }
+
+.user-navigation .nav-pills li a,
+button {
+  border-radius: 0px;
+}
```

Comment:
We don't have anything better than  a and button ? maybe `li > a`  at least?

Reply:
oh yeah! this was a mistake, this should be inside the > li {} block